utils/fetch-meetup.js

```diff
@@ -0,0 +1,49 @@
+const argv = require("yargs").argv;
+const fetch = require("node-fetch");
+const { writeFileSync } = require("fs");
+
+async function main() {
+  const req = await fetch(
+    `https://api.meetup.com/copenhagenjs/events/${argv.id}`
```

Comment:
how do you get this id? I mean when you run the node script, you have `--id=xxxx`, where do you get the id?

Reply:
It is the unique id from meetup that is part of the url, you can't guess it, but it is quicker to grab it than copying the information manually like done before :)